### ISSUE TITLE
[UwU] Improve authors info A11Y

### ISSUE THREAD

**Issue by crutchcorn** [unicorn-utterances/unicorn-utterances#759]
- [ ] Make unicorn picture `alt=""` because it immediately is followed by the name per example 3 of this site: https://www.w3.org/WAI/tutorials/images/decorative/. @evelynhathaway seems to agree

- [ ] Make this a list with a label of "${name} social media links" 

<img width="740" alt="image" src="https://github.com/unicorn-utterances/unicorn-utterances/assets/9100169/4b877801-f3bc-4305-9750-3caa194ccf07">

- [ ] Make "contributor roles" a list with label
- [ ] Remove sticker image alt, as is presentational (@evelynhathaway agrees)

<img width="563" alt="image" src="https://github.com/unicorn-utterances/unicorn-utterances/assets/9100169/ee4c7cd2-aeaa-41cb-8dd9-d2298cd1a5c1">

**Comment by evelynhathaway** [comment]
> Make the pronouns area not an h1 and add an aria-description to explain what it is

Do you mean adding a description or label to the area that says `they/them`? 

I think pronouns appearing right after names are common place enough now that keeping the assistive technology experience close to the visual experience would be preferred. The adage of *No ARIA is better than Bad ARIA* could apply here. ARIA labels and descriptions are often not translated, and a bug could replace the existing accessible name with something less understandable in the future (in the case of a label). Do you agree?

**Comment by crutchcorn** [comment]
I'm admittedly less concerned about translations, as we have a pretty robust translation system (that won't be ready for v1, but we're tracking in #610 and have a strong foundation for), but overall makes sense to not add this.

Will remove from this list - thanks for your feedback!

**Comment by evelynhathaway** [comment]
Sounds good! And just for awareness, I meant automated translations like someone using Google Translate on the page if it's not available in their language.

It's not a WCAG criterion, just a best practice. Visually hidden text (this utility is available in react-aria) can be used instead of `aria-label` to improve this behavior. `aria-labelledby` points to an element in the document that gets translated, so it doesn't have the same issues either.
